### ISSUE TITLE
Most Jbrowses  lack descriptions

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#137]
see #136 for an example description.  Doesn't have to be much, but ideally should include the version of the genome assembly and what the annotations are/where they are from for a genome, and what is the reference genome for transcriptomes mapped to genomes.

**Comment by bradfordcondon** [comment]
J regia is incorrectly listed as F pennsylvanica

**Comment by bradfordcondon** [comment]
For example 
<img width="688" alt="screen shot 2018-02-06 at 12 00 50 pm" src="https://user-images.githubusercontent.com/7063154/35872667-69004dec-0b35-11e8-8e96-55c48aed232b.png">

**Comment by bradfordcondon** [comment]
### J regia English Walnut

The English Walnut (_Juglans regia_) Jbrowse instance corresponds to the [v 1.0 reference genome assembly](http://160.36.205.61:8095/bio_data/1963051).
The Black Walnut (_Juglans nigra_) variants track were generated using GMAP. 

| Total _J. nigra_ mrna contigs |  Contigs mapped to _J. regia_ | >90% identity | >90% coverage |
|:--------------------:|:----------------:|:---------------:|:---------------:|
| 78,834             | 74,984         | 68,235        | 62,387        |



### [Q lobata](http://160.36.205.61:8095/organism/Quercus/lobata) Valley Oak
The Valley Oak (_Quercus lobata_) Jbrowse instance uses the current version 1.0 assembly. Please see [the Genome Assembly page](http://160.36.205.61:8095/bio_data/1963056) for more details.

**Comment by bradfordcondon** [comment]
### Chinese Chestnut

The Chinese chestnut Jbrowse instance corresponds to the [v1.1 reference genome assembly](http://160.36.205.61:8095/bio_data/1962958).  

Additional tracks:

* Chinese chestnut variants and read alignments
* American chestnut variants and read alignments

**Comment by bradfordcondon** [comment]
### Chinese Chestnut QTL

* Augustus, glimmer, maker, maker_repeatmasker tracks
* homolog alignment tracks?
* Read lignment tracks
* Transcript sequence alignment tracks


http://160.36.205.61:8095/Genome-assembly/1962953

**Comment by bradfordcondon** [comment]
@mestato when you have a second can you help me understand hwere the chinese chestnut QTL jbrowse tracks come from?  I cannot find anything about them in the wiki.

**Comment by almasaeed2010** [comment]
Is this resolved? I can't tell if it has been discussed in a meeting

**Comment by bradfordcondon** [comment]
no, we need a description for chinese chesnut qtl.  i've stubbornly refused to figure out what the heck that seperate qtl genome is so i cant just write one up.
